examples/ima-video.amp.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
```

Comment:
Can I somehow enable the amp-ima-video experiment in this file, or will developers have to enable it in their dev consoles?